docs/gettingstarted/manifest-deployer-tutorial/README.md

```diff
@@ -0,0 +1,358 @@
+
+The application itself is not really of interest, it just keeps a container up and running. The container runs in a Pod, the  Pods has 3 Replicas, and all of this is specified in a Deployment manifest which you find in /manifest-deployer-tutorial/manifests/my-deployment.yaml. The other resource manifest we need for the application, is a Kubernetes resource of type Secret. This will allow Kubernetes to access the OCI registry which we deployed in the step "Install the Landscaper together with an OCI registry".
+
+Let us go through the steps, to prepare our Landscaper manifest deployment:
+
+1. In the component-descriptor.yaml (/manifest-deployer-tutorial/component-archive/), replace the `<OCIURL>` placeholder with the OCI registry URL as obtained in section "Install the Landscaper together with an OCI registry". E.g. replace `<OCIURL>` with o.ingress.p.democlust.shoot.k8s.myprovider.com.
+
+2. In the resources.yaml (/manifest-deployer-tutorial/component-archive/), replace the `<IMAGEURL>` placeholder with the container image URL used in section "Push demo image content into OCI registry", to push the sample image into our OCI registry. E.g. replace `<IMAGEURL>` with o.ingress.p.democlust.shoot.k8s.myprovider.com/hello.
+
+3. In deploy-excution.yaml (/manifest-deployer-tutorial/component-archive/blueprint/), replace the `<DOCKERCONFIG>` placeholder with a base64 encoded version of dockers config.json. You already did a docker login into your OCI registry, right?
+
+    ``` bash
+    cat ~/.docker/config.json | base64 --wrap=0
+    ```
+
+4. Add the resources defined in resource.yaml to component-descriptor.yaml by executing
+
+    ``` bash
+    component-cli component-archive resources add ./component-archive ./component-archive/resources.yaml
+    ```
+
+5. Verify updated content of component-descriptor. You need to replace the placeholder `<OCIURL>` with the OCI registry URL obtained in the section "Install the Landscaper together with an OCI registry", e.g. "o.ingress.p.democlust.shoot.k8s.myprovider.com"
+
+    ``` yaml
+    component:
+      componentReferences: []
+      name: test.net/test
+      provider: internal
+      repositoryContexts:
+      - baseUrl: <OCIURL>
+        componentNameMapping: urlPath
+        type: ociRegistry
+      resources:
+      - access:
+          imageReference: <OCIURL>/hello
+          type: ociRegistry
+        name: hello
+        relation: external
+        type: ociImage
+        version: v0.1.0
+      - access:
+          filename: sha256:a250692a6ca416d7d9442e22597eca9f2d5f0ab3dd2af231733ccdc3fb48820a
+          mediaType: application/vnd.gardener.landscaper.blueprint.v1+tar+gzip
+          type: localFilesystemBlob
+        name: blueprint
+        relation: local
+        type: blueprint
+        version: v0.1.0
+      sources: []
+      version: v0.1.0
+    meta:
+      schemaVersion: v2
+    ```
+
+6. Transform component-archive into Component Transport Format for our OCI registry:
```

Comment:
I know this is very confusing and I'm not sure if this is really right but the think the correct description is: Convert the component located in ./component-archive/ into the Component Transport Format (CTF/ just a compressed tar of the folder structure) and add it to a component archive (just a tar of one or more components in CTF) located here ./transport.tar.